### PR TITLE
Leaderboard project setup 

### PR DESCRIPTION
**Pull request by melashu**
### In this milestone, I 

- Installed webpack, webpack dev server, CSS-loader, and style-loader 
- Created an HTML, file structure for the leaderboard app
- Created a CSS style to create medium-fidelity wireframes UI

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    "type": "git",
+    "url": "git+https://github.com/melashu/Leaderboard.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/melashu/Leaderboard/issues"
+  },
+  "homepage": "https://github.com/melashu/Leaderboard#readme"
+}
```
Comment (Munsa1) [inline package.json]: - [x] It is a requirement that scripts like build, start are added in package.json. See the example below.
````json
  "scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
  }
````

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @melashu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 1 above)
Comment (melashu) [reply package.json]: hi @munsa1 

Thank you for your constructive feedback. However, adding scripts like build, and start in package.json is not the part of the requirement. When I use your suggestion `npm run start` can not open the browser. It simply watches the change.

Comment (melashu) [reply package.json]: Here is the requirement 

image

**Review by gemmen29 — APPROVED** [review]
Hi @melashu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**